packages/core/contracts/TestFoundryContract.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
nit: can we put this under a test folder?

Reply:
can. point of this was to demonstrate the use of this. totally can go in a separate directory.

Reply:
Okay, cool, we should definitely do this in a follow up.